.goreleaser-schema.json

```diff
@@ -0,0 +1,2707 @@
+{
+	"$schema": "http://json-schema.org/draft-04/schema#",
+	"$ref": "#/definitions/Project",
+	"description": "goreleaser configuration definition file",
+	"definitions": {
```

Comment:
I wonder if this is static and specific to the `goreleaser` binary. 
Does this need to be checked in / maintained on `talisman` ?

Reply:
@svishwanath-tw 

Done. I pointed my IDE to https://raw.githubusercontent.com/goreleaser/goreleaser/v1.10.2/www/docs/static/schema.json. instead.